dev/io.openliberty.persistence.3.0.thirdparty/resources/org/eclipse/persistence/exceptions/i18n/DBWSExceptionResource.nlsprops

```diff
@@ -27,7 +27,7 @@
 #   (single quote must be coded as one single quote '). 
 #
 # -------------------------------------------------------------------------------------------------
-47000=Cannot locate the "[{0}]" file.
+47000=Could not locate file [{0}]
```

Comment:
Change back to `Cannot locate the "[{0}]" file.`

Reply:
What about "Can not locate the file: [{0}]" ?

Reply:
Yes to "Cannot" and No to the colon. The preferred syntax is "the _variable_ _noun"_ not "_noun_ _variable_". If the quotes are not needed then `Cannot locate the [{0}] file.` with a period at the end.

Reply:
Unfortunately, most EclipseLink messages are written "_noun variable_". This might be a larger change to keep everything similar. Is it feasible to continue with "_noun variable_"?